packages/apps-config/src/endpoints/testing.ts

```diff
@@ -43,7 +43,8 @@ export const testChains: Omit<EndpointOption, 'teleport'>[] = [
     info: 'aleph-testnet',
     providers: {
       'Aleph Zero Foundation': 'wss://ws.test.azero.dev',
-      Dwellir: 'wss://aleph-zero-testnet-rpc.dwellir.com'
+      Dwellir: 'wss://aleph-zero-testnet-rpc.dwellir.com',
```

Comment:
```suggestion
      Dwellir: 'wss://aleph-zero-testnet-rpc.dwellir.com'
```

This should make the linter happy.

Reply:
I have submitted pull requests for [9857](https://github.com/polkadot-js/apps/pull/9857) and [9852](https://github.com/polkadot-js/apps/pull/9852). Could you please approve them? Thank you!